content/en/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Configure RunAsUserName for Windows pods and containers
+content_template: templates/task
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+This page shows how to enable and use the `RunAsUserName` feature for pods and containers that will run on Windows nodes. This feature is meant to be the Windows equivalent of the Linux' `runAsUser`, allowing users to run the container entrypoints with a different username that their default ones.
```

Comment:
s/of the Linux' `runAsUser`/of the Linux-specific `runAsUser` feature/

Reply:
Done